app/set/directives/savedsearches/editor-directive.js

```diff
@@ -67,26 +58,14 @@ function (
                 .then(function (savedSearch) {
                     $location.url('/savedsearches/' + savedSearch.id);
                     $scope.savedSearch = _.clone(savedSearch);
-                    $scope.isOpen.data = false;
+                    $scope.$parent.closeModal();
```

Comment:
`closeModal` is pushed into the scope by modal directive. We use $parent here because this directive has created an isolate scope